src/content/en/2019/compression.md

```diff
@@ -213,32 +220,33 @@ Additionally, the percentage of Brotli compression is higher for third party con
 
-![](../../../static/images/2019/15_Compression/ch15_fig8_lighthouse.jpg)
+![Lighthouse Compression Suggestions](../../../static/images/2019/15_Compression/ch15_fig8_lighthouse.jpg)
 
-Because the HTTP Archive runs Lighthouse audits for each mobile page, we can aggregate the scores across all sites to learn how much opportunity there is to compress more content. Overall, 62% of websites are passing this audit and almost 23% of websites have scored below a 40. This means that over 1.2 million websites could benefit from enabling additional text based compression.
+Because the [HTTP Archive runs Lighthouse audits](./methodology) for each mobile page, we can aggregate the scores across all sites to learn how much opportunity there is to compress more content. Overall, 62% of websites are passing this audit and almost 23% of websites have scored below a 40. This means that over 1.2 million websites could benefit from enabling additional text based compression.
 
-![](../../../static/images/2019/15_Compression/ch15_fig9_lighthouse_compression_scores.jpg)
+!["Enable Text Compression" Lighthouse Scores](../../../static/images/2019/15_Compression/ch15_fig9_lighthouse_compression_scores.jpg)
 
 Lighthouse also indicates how many bytes could be saved by enabling text compression. Of the sites that could benefit from text compression, 82% of them can reduce their page weight by up to 1 MB!
 
-![](../../../static/images/2019/15_Compression/ch15_fig10_lighthouse_compression_byte_savings.jpg)
+!["Enable Text Compression" Potential Byte Savings](../../../static/images/2019/15_Compression/ch15_fig10_lighthouse_compression_byte_savings.jpg)
+
+## Conclusion
 
-# Conclusion
+HTTP Compression is a widely used and highly valuable feature for reducing the size of web content. Both gzip and brotli compression are the dominant algorithms used, and the amount of compressed content varies by content type. Tools like Google Lighthouse can help uncover opportunities to compress content.
 
-HTTP Compression is a widely used and highly valuable feature for reducing the size of web content. Both gzip and brotli compression are the dominant algorithms used, and the amount of compressed content varies by content type. Tools like Google Lighthouse can help uncover opportunities to compress content. 
+While many sites are making good use of HTTP compression, there is still room for improvement, particularly for the text/html format that the web is built upon! Similarly, lesser understood text formats like font/ttf, application/json, text/xml, text/plain, image/svg+xml, and image/x-icon may take extra configuration that many websites miss.
 
 At a minimum, websites should use gzip compression for all text based resources, since it is widely supported, easily implemented and has a low processing overhead. Additional savings can be found with brotli compression, although compression levels should be chosen carefully based on whether a resource can be precompressed.
```

Comment:
Hyphen
Comma
```suggestion
At a minimum, websites should use gzip compression for all text-based resources, since it is widely supported, easily implemented, and has a low processing overhead. Additional savings can be found with brotli compression, although compression levels should be chosen carefully based on whether a resource can be precompressed.
```